scripts/src/main/resources/scripts/command/ide

```diff
@@ -236,10 +243,74 @@ function doSetup() {
   doUpdateCli
   doUpdateMetaFile
   doUpdateSettings "${@}"
+  local current_devon_version
+  local min_devon_ide_version_month
+  local current_devon_version_month
   if [ ! -f "${SETTINGS_PATH}/devon.properties" ]
   then
     doEcho "Your settings are missing the 'devon.properties' file. Please ask your technical lead to properly merge and update your settings."
     doDownload "https://raw.githubusercontent.com/devonfw/ide-settings/master/devon.properties" "${SETTINGS_PATH}"
+  else
+    current_devon_version="$("${DEVON_HOME_DIR}/scripts/devon.bat" -v)"
+    min_devon_ide_version_month=${DEVON_IDE_MIN_VERSION##${DEVON_IDE_MIN_VERSION%%.*}.} 
+    current_devon_version_month=${current_devon_version##${current_devon_version%%.*}.} 
+    if [[ ${DEVON_IDE_MIN_VERSION%%.*} -gt ${current_devon_version%%.*} ]]
```

Comment:
we should use `doVersionCompare` for this as `-gt` is not suitable for comparing versions. In the worst case, we later have to change versioning schema again (see devonfw Teams discussion) and this will break and stop working.